gui/functionality/add_cluster.py

```diff
@@ -124,26 +146,34 @@ def add_cluster(self):
             save_environment()
 
     def open_actions_file_dialog(self):
+        """ Opens the winodw dialog that allows to select an actions file.
+        """
         self.actions_file_name, _ = QFileDialog.getOpenFileName()
         if self.actions_file_name:
             self.ui.selected_file_path_browser.setText(self.actions_file_name)
             self.ui.selected_file_path_browser.setEnabled(True)
             self.ui.clear_actions_file_button.setEnabled(True)
 
     def clear_actions_file(self):
+        """ Removes the selection of actions file.
```

Comment:
The actions file? I'm not sure

Reply:
Why? The method name is clear_actions_file so it clearly removes an action file.

Reply:
Me too, I added it